winui/AvatarView/Overview.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: Overview of WinUI AvatarView control | Syncfusion
+description: Learn all about the introduction of Syncfusion WinUI AvatarView Control (SfAvatarView) with essential features and more here.
+platform: WinUI
+control: SfAvatarView
+documentation: ug
+---
+
+# WinUI Avatar View (SfAvatarView) Overview
```

Comment:
@MadhumithanJayapathi WinUI AvatarView Overview

Reply:
@Manivannan-E Changed the title as mentioned.